pkg/kv/kvserver/testdata/echotest/TestMultiSSTWriterInitSST

```diff
@@ -1,20 +1,21 @@
 echo
 ----
 >> sst0:
-rangedel: /Local/RangeID/0/r"{"-\x00"}
-rangedel: /Local/RangeID/0/{r"\x00"-s""}
+rangedel: /Local/RangeID/0/{r""-s""}
+rangekeyset: /Local/RangeID/0/{r""-s""} -> /<empty>
 rangekeydel: /Local/RangeID/0/{r""-s""}
 >> sst1:
-rangedel: /Local/Range"d"{-}
 rangedel: /Local/Range{"d"-/Max}
+rangekeyset: /Local/Range{"d"-/Max} -> /<empty>
 rangekeydel: /Local/Range{"d"-/Max}
 >> sst2:
-rangedel: /Local/Lock/{Local/Range"d"-""\x12\x01k\x12d\x00\xff\x01\x00\x01\x00""}
-rangedel: /Local/Lock/{""\x12\x01k\x12d\x00\xff\x01\x00\x01\x00""-Local/Range/Max}
+rangedel: /Local/Lock/Local/Range{"d"-/Max}
+rangekeyset: /Local/Lock/Local/Range{"d"-/Max} -> /<empty>
 rangekeydel: /Local/Lock/Local/Range{"d"-/Max}
 >> sst3:
-rangedel: /Local/Lock{"d"-/""\x12d\x00\x01\x00""}
-rangedel: /Local/Lock/{""\x12d\x00\x01\x00""-Max}
+rangedel: /Local/Lock{"d"-/Max}
+rangekeyset: /Local/Lock{"d"-/Max} -> /<empty>
 rangekeydel: /Local/Lock{"d"-/Max}
```

Comment:
What does it mean that these two `rangekey` things overlap by the span? What is the precedence between them?

Reply:
My understanding is that rangedels and rangekeys are different. Rangedels clear points in their span, whereas rangekey are MVCC deletion markers for a key span. Furthermore, range keys can "stack", since you can have overlapping ones at different timestamps (the timestamps aren't really visible here because I'm not putting any). Due to fragmentation, we end up with exact overlapping copies.

Btw, I'm not sure I see your comment in the right place, but these are the general thoughts that hopefully explain what you were looking at.